code/modules/surgery/organs/external/tails.dm

```diff
@@ -78,6 +78,9 @@
 	color_source = ORGAN_COLOR_HAIR
 	wag_flags = WAG_ABLE
 
+/obj/item/organ/external/tail/cat/get_global_feature_list()
+	return GLOB.tails_list_human
```

Comment:
theres a global tail list in the parent type organ/external/tail

Reply:
There is but it would assign lizards a cat tail and felinids a lizard tail every now and then because it contains all tails.